src/lib/download/types/ParameterQualifiersHeader.ts

```diff
@@ -0,0 +1,4 @@
+export interface ParameterQualifiersHeader {
```

Comment:
Does `@deltares/fews-pi-requests` not have an equivalent type? Maybe you can use [`Pick`](https://www.typescriptlang.org/docs/handbook/utility-types.html#picktype-keys) to extract a subset of this type, to guarantee that the types remain consistent.